src/safe-upgrade-tool/install.md

```diff
@@ -0,0 +1,100 @@
+---
+group: software-update-guide
+title: Install
+ee_only: True
+functional_areas:
+  - Upgrade
+---
+
+The Safe Upgrade Tool ALPHA (SUT) is a command line (CLI) tool that checks a Magento instance against a specific version by analyzing all the non-Magento modules installed on it.
+
+{:.bs-callout-warning}
+At the moment this is an ALPHA version with limited scope, only validating PHP Magento APIs and GraphQL schema.
+
+## Workflow
+
+The following diagram shows the expected workflow when running the SUT:
+
+![SUT Diagram](img/mvp-diagram.png)
+
+### Who is the SUT for?
+
+The following use case describes the typical process for a Magento partner to upgrade a client's Magento instance:
+
+1. A partner's Software Engineer downloads the SUT and executes it during the beta phase of the newest Magento release.
+1. The Software Engineer sees that there are several customized areas broken in the inventory and catalog modules and they also get a complexity score of X. See the [Developer information guide]({{site.baseurl}}/safe-upgrade-tool/developer.html) for more information on the complexity score.
+1. With this information, the Software Engineer is able to understand the complexity of the upgrade and is able to relay this information back to the partner's Account Manager.
+1. The Account Manager creates a timeline and cost for the Magento upgrade, which allows them to get their manager's approval.
+1. With their manager's approval, the Software Engineer works on the required code modifications to fix the broken modules.
+1. The Software Engineer runs the SUT tool one more time with a Magento pre-release to ensure there are no new issues and that their code changes fixed the problems found during the beta phase.
+1. Everything check’s out and the Software Engineer pushes the code to a staging environment where regression tests confirm all tests are green, which allows them to release the latest Magento version to production the same day that the Magento pre-release is released.
+
+## Prerequisites
+
+See [prerequisites]({{site.baseurl}}/safe-upgrade-tool/prerequisites.html) for more information.
+
+{:.bs-callout-info}
+You can run the SUT in any operating system. There is no requirement to run the SUT where your Magento instance is located. It is necessary for SUT to have access to the source code of the Magento instance. For example, you can install the SUT on one server and point it at your Magento installation on another server.
+
+### Recommended actions
+
+It is recommended to avoid naming SUT modules with the same name, as a segmentation fault error can occur.
+
+If this segmentation error takes place, solve it by:
+
+*  Avoid analyzing a set of modules if there are 2 modules with the same name.
+*  Analyze modules separately with the option -m:
+
+    ```bash
+    bin/sut upgrade:check /(instance_path) --coming-version=2.4.1 -m /(module_path)
+    ```
+
```

Comment:
Please add:
"If you get memory issues while executing SUT our recommendation is to use the -m command to run the tool against a specific module, this will prevent the problem to happen"